### DISCUSSION TITLE
Ghost Client OutlookV2 not sending attachments, assuming old Redemption version in the folder

### DISCUSSION THREAD

**Discussion by theusername-sudo** [cmu-sei/GHOSTS#269] [General]
I've been doing some testing and when I use outlookv2 handler in my timeline it seems to not send any attachments, when I update with a the newest Redemeption64.dll, [Redemption Site for the updated versions](https://www.dimastr.com/redemption/download.htm) it functions well but the problem is they only hand out the Dev version which requires EULA acceptance at every run time. I can provide pictures if needed.

**Comment by sei-dupdyke** [comment]
Hi, thanks for checking out the project. Our advise would be that you should probably purchase that version if you're going to use it in a production environment please — redemption is awesome software, please support them!

**Comment by theusername-sudo** [comment]
Could you tell me which version of Redemption is currently used in 7.2.0?

**Reply by sei-dupdyke** [reply]
Yep, I have 5.6.0.3644.

**Comment by theusername-sudo** [comment]
Thank you!
